pkgs/by-name/id/ida-free/srcs.json

```diff
@@ -0,0 +1,14 @@
+{
+    "x86_64-linux": {
+        "url": "https://out7.hex-rays.com/files/idafree84_linux.run",
+        "sha256": "1wg60afkhjj7my2la4x4qf6gdxzl2aqdbvd6zfnwf8n3bl7ckn2a"
```

Comment:
Looks like the package will be broken due to hash mismatch once upstream releases something like 8.4.240327?

Reply:
I didn't think of this actually, but you should be correct. At a first glance, they don't seem to offer pinned downloads. Do you have an idea on how to handle this?

Reply:
web.archive.org cam solve this